src/util/FormUtils.js

```diff
@@ -222,6 +222,9 @@ export default class FormUtils {
     if (question[Constants.PATTERN]) {
       return true;
     }
+    if (this.containsXSDProperty(question)) {
```

Comment:
This is a **little** strange ... why do you need to use the method
`hasValidationLogic`

I thought the validator itself should know if he can be applied or not and I would not separate this logic outside of it.

Reply:
I do not need, it is just for optimization... So that it does not have to go through the validators if there is no need in the first place. I can remove it if if we prefer to have really only the validator factory take care of it.

Reply:
It does not have any other usage in s-forms, if we agree I can completely remove it.

Reply:
We agreed that this will be taken care by #312